docs/source/project_documentation/specifications.md

```diff
@@ -0,0 +1,92 @@
+# Specifications Plugin
+
+The `Specifications` plugin traces `Requirement`s of provided `Specification`s.  It does this through the `spec-csv` command, which generates a .csv file for a `Specification` which tracks its `Requirement`s.
```

Comment:
So, remember the code highlighting of `using the back ticks` is only necessary if you are referencing code components. 

```suggestion
The Specifications Plugin traces requirements of provided modeled specifications.  It does this through the `spec-csv` command, which generates a *.csv file for a `req-spec` which traces its `requirements`.
```